sci-libs/cantera/files/cantera_2.4.0_env.patch

```diff
@@ -0,0 +1,33 @@
+diff -Nur old/cantera-2.4.0/SConstruct new/cantera-2.4.0/SConstruct
+--- old/SConstruct	2018-08-24 16:24:45.000000000 +0300
++++ new/SConstruct	2019-06-22 00:18:41.000000000 +0300
+@@ -189,7 +189,7 @@
+     toolchain = ['default']
+ 
+ env = Environment(tools=toolchain+['textfile', 'subst', 'recursiveInstall', 'wix', 'gch'],
+-                  ENV={'PATH': os.environ['PATH']},
```

Comment:
Can't you, say, `ENV=os.environ`? ;-)

Reply:
Do you mean the replacing of `ENV={'PATH': os.environ['PATH'], 'CCACHE_DIR': os.environ.get('CCACHE_DIR','')}` with `ENV=os.environ`? 

This change doesn't affect working of ccache but corrupts 2 tests because some test input files will not be found (internal CANTERA_DATA path variable seems to be broken this way).

I experimented during previous package introducing pull request and the current ENV assignment makes working both tests and ccache.

Reply:
That is weird but *shrug*.

Reply:
The current solution works and sufficiently well tested.
I'm afraid the attempts to change it will be a bargain one trouble for another.